### PR TITLE
Support changed playwright api

### LINKED ISSUE #21: Support changed playwright api
breaking changes since https://github.com/microsoft/playwright-python/releases/tag/v1.8.0a1

### PR DESCRIPTION
**Pull request by pashidlos**
closes: https://github.com/Visual-Regression-Tracker/sdk-python/issues/21

### CODE REVIEW

**Comment by pashidlos** [comment]
@dcrowe could you assist with failed test?
- python 3.7 has no `Literal` type
- seems like the last one is always failing

**Comment by pashidlos** [comment]
heh, fixed now )

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
 requests
 dacite
-playwright; python_version >= '3.7'
+playwright >= 1.10; python_version >= '3.7'
 importlib-metadata; python_version <= '3.7'
 dataclasses; python_version <= '3.6'
 greenlet; python_version <= '3.6'
+typing-extensions
```
Comment (dcrowe) [inline requirements.txt:7]: Should this extension only be added for <=3.6

Comment (dcrowe) [reply requirements.txt:7]: Nevermind, I see how you used it.

**Review by dcrowe — APPROVED** [review]

**Comment by pashidlos** [comment]
@dcrowe thanks!

**Pull request merged**